### DISCUSSION TITLE
nb_lookup api filter

### DISCUSSION THREAD

**Discussion by jk-onnet** [netbox-community/ansible_modules#586] [Q&A] [answered]
May I ask how to filter the cluster object by cluster type? In NetBox API, I can define the apifilter with json, but I'm not sure how to do with ansible debug msg.
![image](https://user-images.githubusercontent.com/85537520/132796670-30cd09e3-5cfb-44fc-8790-67dfe0aaa14f.png)

**Comment by ryanmerolle** [comment, marked as answer]
```
"{{ query('netbox.netbox.nb_lookup', 'clusters', api_endpoint=netbox_url, api_filter='type=newcluster',  token=netbox_token) }}"
```

The above should work, remember that you should `reference type=<type.slug>`
